components/automate-chef-io/Makefile

```diff
@@ -38,8 +41,13 @@ sync_swagger_files: clean_swagger_files
 
 $(SWAGGER_RESULT_FILE): $(SWAGGER_FILES)
 
+#generate_swagger: $(SWAGGER_RESULT_FILE)
+#	swagger mixin --quiet -c=$(EXPECTED_CONFLICTS) $(SWAGGER_FILES) > $(SWAGGER_RESULT_FILE)
+
 generate_swagger: $(SWAGGER_RESULT_FILE)
-	swagger mixin --quiet -c=$(EXPECTED_CONFLICTS) $(SWAGGER_FILES) > $(SWAGGER_RESULT_FILE)
+	swagger mixin $(STATIC_SWAGGER_FILES) $(SWAGGER_FILES) | \
+		jq '.paths = (.paths | with_entries( select( all( .value[].tags[]; . != "hidden") ) ) )' \
```

Comment:
In English, this says to replace the `paths` object in the swagger doc with a new copy that has any endpoint with a tag named "hidden" removed. 

The swagger path to tags looks generally like this:

```
{
  ...
  "paths": {
    "/path/to/my/endpoint": {
       "HTTP_METHOD": {
         ...
         "tags": ["foo", "bar"],
```

Reply:
👏 nicely done